query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/batch/transactional_batch.rs

```diff
@@ -100,7 +101,7 @@ mod transactional {
         Ok(())
     }
 
-    #[connector_test(exclude(MongoDb))]
+    #[connector_test(exclude(MongoDb, TiDB))]
```

Comment:
@Mini256 ?

Reply:
Also because TiDB does not support serializable transaction isolation level at present, I created a special test case for it: https://github.com/prisma/prisma-engines/pull/3308/commits/c4469be03df14e6c0e4bddadb949c7fe83d5e7fe